.github/workflows/linters.yml

```
@@ -50,7 +49,7 @@ jobs:
       - name: Lint C/C++ files
         run: make lint-cpp
   format-cpp:
-    if: ${{ github.event.pull_request.draft == false && github.base_ref == 'master' }}
+    if: ${{ github.event.pull_request.draft == false && github.base_ref == 'main' }}
```

Could we remove the `github.base_ref == 'main'` check? @RaisinTen 

```suggestion
    if: github.event.pull_request.draft == false
```

---

No, we don’t want this check to run on backport PRs or get in the way of the release process. However I’m pretty sure we don’t need to put the actual branch name, IIRC GitHub gives us a `default_branch` property that we use on some other workflow.